apps/src/assetManagement/ImagePicker.jsx

```diff
@@ -0,0 +1,71 @@
+var AssetManager = require('./AssetManager.jsx');
+var IconLibrary = require('./IconLibrary.jsx');
+
+/**
+ * A component for managing hosted assets.
+ */
+module.exports = React.createClass({
+  propTypes: {
+    assetChosen: React.PropTypes.func,
+    typeFilter: React.PropTypes.string,
+    channelId: React.PropTypes.string.isRequired,
+    uploadsEnabled: React.PropTypes.bool.isRequired
+  },
+
+  getInitialState: function () {
+    return {mode: 'files'};
+  },
+
+  setIconMode: function () {
+    this.setState({mode: 'icons'});
+  },
+
+  setFileMode: function () {
+    this.setState({mode: 'files'});
+  },
+
+  render: function () {
+
+    var modeSwitch, title = this.props.assetChosen ?
+      <p className="dialog-title">Choose Assets</p> :
+      <p className="dialog-title">Manage Assets</p>;
+
+    if (this.props.assetChosen) {
+      var fileMode = this.state.mode === 'files';
+      modeSwitch = <div>
+        <p onClick={this.setFileMode} style={{
+          float: 'left',
+          margin: '0 20px 0 0',
+          fontFamily: fileMode ? '"Gotham 5r"' : null,
```

Comment:
This needs to be in nested quotes rather than just `fontFamily: fileMode ? 'Gotham 5r' : null` ?